telosnetwork/telos-wallet
# adding new Telos cloud submenu

# Fixes #581

## Description

This PR refactored the HomePage making the Telos Cloud Button much more visible and highlighted, which in turn, when pressed redirects the user to a second sub nemu of web2 oauth provier options. 

## Test scenarios
- Go to [this link](https://deploy-preview-613--wallet-staging.netlify.app/)
- The login buttons should look like the screenshot
- Press the first button
  - A second submenu should appear replacing the previous one
  - Only the first button is enabled
  - The last two buttons should be and look disabled (with no rendering problems)
  - If you press back button it should redirect you back to home menu
  - If you press Sign with Google the OreId popup will show (remember this won't work on netlify links)




## Screenshots

![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/5c8420b9-3adf-43a3-b88d-2e542c69b34f) ![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/d997c091-19c3-4172-90ef-205f5d6b9546) 




## Checklist:
